extensions/commons/README.md

```diff
@@ -45,7 +45,7 @@ An incomplete Collection:
 ```
 {
   "stac_version": "0.9.0-rc1",
-  "stac_extensions": ["commons"],
+  "stac_extensions": ["commons", "eo", "view"],
```

Comment:
This change needs a revert, only changes required in the other two examples.

Reply:
ok, was just going to ask that.

Reply:
https://github.com/radiantearth/stac-spec/pull/734/commits/d9a04acadcec64e3fc9fefb595f87a0e794ad4e9